### PR TITLE
Vox have a proper monkey form

### PR DESCRIPTION
**Pull request by ghost**
It is a small green chicken, can be butchered for raw chicken, and picked up

Vox chicken

Copypasted some of intigracy's code and modified it a bit so masks and hats aren't floating above them as much, but it still looks fairly choppy

Vox chicken nurse

Unsure of what else to add to make them actually unique from the default monkey
- [x] Add green eggs (their version of monkey cube, when incubated = vox chicken)
- [x] Remove their hands
- [x] Change their attack text to pecking
- [x] Add some actual features to them beyond them being a resprited monkey

### CODE REVIEW

**Comment by ghost** [comment]
Those phantom commits are gonna haunt me forever, aren't they?

**Comment by Probe1** [comment]
yes

**Comment by RogerWilcoBeep** [comment]
You feel it, too, don't you?
 I'm gonna make them give back our memes.

**Comment by clusterfack** [comment]
Phantom commits don't matter anymore, as it says in the IRC we have squash merging now (as you can see by checking the commit log)

**Comment by Probe1** [comment]
@dragonbro
did you make a vox monkeycube and voxcube container?  make it orderable from cargo too.

**Comment by Intigracy** [comment]
You could make them not able to pick things up or take things off that they're wearing. People forcefully dressing up chickens when?

If you do this though, do remove the nitrogen breath requirement for obvious reasons.

**Comment by SkowronX** [comment]
also make a spiked sprite that gives you chicken.

**Comment by ghost** [comment]
Got some suggestions from deadchat, updating the stuff to do list

**Comment by Intigracy** [comment]
Actually Skowron has a point, since they're a monkey subtype you can't just butcher them for chicken, and if you throw them on the spike as-is you'll get the monkey sprite with your chicken that it drops.

Unless spikes do some nice icon handling and just use the icon of the mob you're throwing on it (like rechargers), but I doubt it.

**Comment by Shadowmech88** [comment]
> since they're a monkey subtype you can't just butcher them for chicken

Does butchering not use the `meat_type` var?

**Comment by Intigracy** [comment]
I edited my comment after realizing you set that, derp.

Though you might wanna double check just to be safe.

**Comment by ghost** [comment]
Already tested butchering, you get 2 chicken from knife butchering, then a final third from spiking it, so the meat_type var does indeed work with both butchering and spiking.

As the spike uses the base monkey sprite for diona nymph spiking too, it seems that it isn't a case of nice icon handling (Although, there is a unique sprite for spiking a xeno)

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 1]
```diff
@@ -699,7 +699,4 @@ var/global/list/animal_count = list() //Stores types, and amount of animals of t
 				adjustBruteLoss(volume * 0.5)
 		if("pacid")
 			if(!supernatural)
-				adjustBruteLoss(volume * 0.5)
-
-
-/datum/locking_category/simple_animal
+				adjustBruteLoss(volume * 0.5
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/simple_animal.dm]: Welp, think I fucked up rebase, because I do not know how this got here

**Comment by ghost** [comment]
Holy shit I am apparently retarded, hang on

File: code/modules/mob/living/carbon/monkey/monkey.dm [hunk 2]
```diff
@@ -10,6 +10,7 @@
 	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/animal/monkey
 	species_type = /mob/living/carbon/monkey
 	treadmill_speed = 0.8 //Slow apes!
+	var/attack_text = "bites"
```
Comment (Intigracy) [inline code/modules/mob/living/carbon/monkey/monkey.dm:13]: Thanks, missed monkeys.

File: code/modules/reagents/reagent_containers/food/snacks/egg.dm [hunk 3]
```diff
@@ -71,3 +72,15 @@
...
 	else
 		..()
+
+/obj/item/weapon/reagent_containers/food/snacks/egg/vox
+	name = "green egg"
+	desc = "Looks like it came from some genetically engineered chicken"
+	icon_state = "egg-vox"
+
+
+	hatch()
```
Comment (Intigracy) [inline code/modules/reagents/reagent_containers/food/snacks/egg.dm:82]: b a d

File: code/modules/mob/living/carbon/monkey/monkey.dm (hunk 2 above)
Comment (ghost) [inline code/modules/mob/living/carbon/monkey/monkey.dm:13]: Seems it didn't do anything anyway, still shows as 'bit'

File: code/modules/reagents/reagent_containers/food/snacks/egg.dm (hunk 3 above)
Comment (ghost) [inline code/modules/reagents/reagent_containers/food/snacks/egg.dm:82]: I asked in the IRC, but all I got was memes

Comment (Probe1) [inline code/modules/reagents/reagent_containers/food/snacks/egg.dm:82]: I completely believe that.

Comment (Intigracy) [inline code/modules/reagents/reagent_containers/food/snacks/egg.dm:82]: never partial path is the answer for the future fyi

**Comment by Intigracy** [comment]
woooooooooooooooooow

**Comment by ghost** [comment]
Welp

**Comment by ghost** [comment]
Does a similar thing occur with monkeys?

Also, I noticed that there isn't a way to declare a dead icon state, so if that's just flipping the icon by 90 degrees, how is it off center?

**Comment by SkowronX** [comment]
So uh can you give vox chickens the more updated monkey ui and have usable "hands" when given methylin? You know, so it's fun to play.

**Comment by ghost** [comment]
Working on that now, would've been nice to have such things said before it got merged, also what do you mean by 'more updated monkey ui'?

**Comment by ihadtoregisterforthis** [comment]
Vox chickens die in vox-only environments. Which is as bad as stealing forty cakes.

**Comment by ghost** [comment]
Aw shit. Guess that's what happens when you make them based entirely off monkeys.

They're literally just playable chickens at this point, still collecting info on what to fix for them, thanks for pointing that out though.

**Comment by Shadowmech88** [comment]
Proper species datums for primitive forms would be nice.

**Comment by ihadtoregisterforthis** [comment]
They can't *flap or *aflap. I suggest revert until this critical elements are added.
They cant either eat wheat by themselves.

**Comment by ghost** [comment]
@ihadtoregisterforthis Yeezus christ, this is horrible

I shall fix this post haste

**Comment by ghost** [comment]
Okay, new list of shit to fix
- Add flap and aflap emotes
- Add it so that vhickens can peck at wheat/corn
- Add methylin check to the hands
- Figure out a way to make it so vhickens only have the one hand (Their beak)
- Make them a balance between the two extremes, they can survive in a N2 only environment, but they don't get poisoned by O2
-  Figure out what Skowron meant by 'more updated monkey UI'

**Comment by Shadowmech88** [comment]
If you could just make complete species datums for all primitive forms, that'd be _great_.

**Pull request merged**